### PR TITLE
fix(DB/onkill_reputation) Venture Co. mobs in stv

### PR DESCRIPTION
**Pull request by BeefSteaker**
## Changes Proposed:
-  Updates on kill rep for several Venture Co. mobs in stv
-  Sets Venture Co. Lumberjack to not give rep after honored

## Issues Addressed:

- Closes [Venture Co. mobs in stv give too much reputation #13586 ](https://github.com/azerothcore/azerothcore-wotlk/issues/13586)
- Closes [Venture Co. mobs in stv give too much reputation #4324 ](https://github.com/chromiecraft/chromiecraft/issues/4324)

## Tests Performed:

- killed mobs on local
- 

## How to Test the Changes:

1. .go c id ( 675, 1096, 1097, 1094, 4260, 674, 676, 677, 1095, 4723)
2. kill them
3. note xp gain (account for human bonus)

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by heyitsbench** [comment]
Tested a few mobs. [Venture Co. Tinkerer](https://user-images.githubusercontent.com/38561765/199383829-93b3279b-bc58-458b-ae7e-a9f9b391c3ff.png), [Venture Co. Strip Miner](https://user-images.githubusercontent.com/38561765/199383867-12b54717-4e1a-4520-a479-b0eca9dab5e7.png), and got nothing for Venture Co. Surveyors.

**Comment by acidmanifesto** [comment]
> Tested a few mobs. [Venture Co. Tinkerer](https://user-images.githubusercontent.com/38561765/199383829-93b3279b-bc58-458b-ae7e-a9f9b391c3ff.png), [Venture Co. Strip Miner](https://user-images.githubusercontent.com/38561765/199383867-12b54717-4e1a-4520-a479-b0eca9dab5e7.png) ([Aiming for this](https://user-images.githubusercontent.com/38561765/198680486-18c2ca44-7c14-44b9-8339-5c413555fbb1.png)), and got nothing for Venture Co. Surveyors.

Working as intended then?

**Comment by heyitsbench** [comment]
> Working as intended then?

No.

File: data/sql/updates/pending_db_world/rev_1667348648920858600.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+--
+UPDATE `creature_onkill_reputation` SET `RewOnKillRepValue1` = 5 WHERE `creature_id` IN (674, 675, 676, 677, 1094, 1095, 1096, 1097, 4260, 4723);
+UPDATE `creature_onkill_reputation` SET `MaxStanding1` = 5 WHERE `creature_id` IN  (921);
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1667348648920858600.sql:3]: Being informed it is not working as intended.
https://github.com/azerothcore/azerothcore-wotlk/pull/13644#issuecomment-1304866420

**Review by acidmanifesto — CHANGES_REQUESTED** [review]

**Review by acidmanifesto — APPROVED** [review]

**Comment by acidmanifesto** [comment]
@heyitsbench up for a re-test?

**Comment by heyitsbench** [comment]
> @heyitsbench up for a re-test?

Not gonna be able to re-test all day unfortunately. ☹️

**Comment by heyitsbench** [comment]
Tested again, here's [Venture Co. Surveyor](https://user-images.githubusercontent.com/38561765/200605707-497adb28-f39d-4f3f-99a1-f707d7dc202f.png), [Venture Co. Strip Miner](https://user-images.githubusercontent.com/38561765/200605639-a838d5df-9f4d-43fa-a65a-ab3ddd3d8727.png), and [Venture Co. Tinkerer](https://user-images.githubusercontent.com/38561765/200605812-748736aa-9110-46e0-8193-723b605685df.png). They should all be giving 5,2,2,2 to my knowledge, I have no idea what's causing the strip miner to give 5,3,3,3 though.

**Comment by BeefSteaker** [comment]
weird, thanks for the test though

**Comment by BeefSteaker** [comment]
On further testing, this seems to apply to all mobs that award 5 xp. they alternate 5222 5333 . could be a rounding thing with it giving 2.5 rep

File: data/sql/updates/pending_db_world/rev_1667348648920858600.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+--
+DELETE FROM `creature_onkill_reputation` WHERE (`creature_id` = 676);
+INSERT INTO `creature_onkill_reputation` (`creature_id`, `RewOnKillRepFaction1`, `RewOnKillRepFaction2`, `MaxStanding1`, `IsTeamAward1`, `RewOnKillRepValue1`, `MaxStanding2`, `IsTeamAward2`, `RewOnKillRepValue2`, `TeamDependent`) VALUES
+(676, 21, 0, 5, 0, 5, 0, 0, 0, 0);
+UPDATE `creature_onkill_reputation` SET `RewOnKillRepValue1` = 5 WHERE `creature_id` IN (674, 675, 677, 1094, 1095, 1096, 1097, 4260, 4723);
+UPDATE `creature_onkill_reputation` SET `MaxStanding1` = 5 WHERE `creature_id` IN  (921);
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1667348648920858600.sql:6]:
```suggestion
UPDATE `creature_onkill_reputation` SET `MaxStanding1` = 5 WHERE `creature_id` IN (921);
```

**Review by Nefertumm — APPROVED** [review]

**Comment by Nefertumm** [comment]
Yes, I remember something about reputation rounding a long time ago, but I don't remember if it was correct or not. However, it's not related to this PR.

**Comment by BeefSteaker** [comment]
Ok, ignoring that I think it's good to merge then

**Pull request merged**